.circleci/config.yml

```diff
@@ -0,0 +1,85 @@
+---
+# Note: CC_TEST_REPORTER_ID, the env var required to upload test coverage to Code Climate, is set from the CircleCI web
+# interface as a private environment variable.
+# https://app.circleci.com/settings/project/github/RITlug/teleirc/environment-variables
+
+# https://circleci.com/docs/2.0/configuration-reference
+version: 2.1
+
+workflows:
+  main:
+    jobs:
+      - go_1-15
+      - go_1-16
+      - go_1-17
+      - build_docs
+
+commands:
+  golintci-lint:
+    description: Run linter checks on TeleIRC.
+    steps:
+      - checkout
+      - run: 
+          name: Download and install golintci-lint.
+          command: curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sudo sh -s -- -b $(go env GOPATH)/bin v1.43.0
```

Comment:
This is sketchy, lol.

Is there some kind of package we can install instead?

Reply:
@xforever1313 Sadly, no… this is literally upstream's recommended installation method. 😭 

https://golangci-lint.run/usage/install/#ci-installation

Reply:
Gross, lol.

I guess one nice thing about using Circle, its not our machine that will get hacked if something sketchy happens :D.